website/docs/cloud-docs/integrations/kubernetes/index.mdx

```diff
@@ -1,68 +1,163 @@
 
-## Prerequisites
+## Key benefits
 
-All Terraform Cloud users can use the Terraform Cloud Operator for Kubernetes. Some features of Terraform Cloud that are [limited to certain tiers](/terraform/cloud-docs/overview) aren't available to the Terraform Cloud Operator for Kubernetes unless you've purchased the corresponding tier.
+The Terraform Cloud Operator v2 offers several improvements over v1, such as:
 
-## Terraform Cloud Operator for Kubernetes
+- Newer TODO
+- Better
+- Faster
 
-### Networking Requirements
+## Supported Terraform Cloud Features
 
-In order for the Terraform Cloud Operator for Kubernetes to function properly, it must be able to make outbound requests over HTTPS (TCP port 443) to the Terraform Cloud application APIs. This may require perimeter networking as well as container host networking changes, depending on your environment. The IP ranges are documented in the [Terraform Cloud IP Ranges documentation](/terraform/cloud-docs/architectural-details/ip-ranges). The services which run on these IP ranges are described in the table below.
+The Terraform Cloud Operator for Kubernetes allows you to create agent pools, deploy modules, and manage workspaces through Kubernetes controllers. These controllers enable you to automate and manage Terraform Cloud resources using custom resources in Kubernetes.
 
-| Hostname         | Port/Protocol  | Directionality | Purpose                                                                                                            |
-| ---------------- | -------------- | -------------- | ------------------------------------------------------------------------------------------------------------------ |
-| app.terraform.io | tcp/443, HTTPS | Outbound       | Dynamically managing Terraform Cloud Workspaces and returning the output to Kubernetes via the Terraform Cloud API |
+### Agent pool
 
-### Compatibility
+Agent pools in Terraform Cloud manage the execution environment for Terraform runs. The Terraform Cloud Operator allows you to create and manage agent pools as part of your Kubernetes infrastructure.
 
-The current release of the Terraform Cloud Operator for Kubernetes supports the following versions:
+The following example creates a new agent pool with the name `agent-pool-development` and generates an agent token with the name `token-red`.
 
-* Helm 3.0.1 and above
-* Kubernetes 1.15 and above
+```yaml
+---
+apiVersion: app.terraform.io/v1alpha2
+kind: AgentPool
+metadata:
+  name: this
+spec:
+  organization: kubernetes-operator
+  token:
+    secretKeyRef:
+      name: tfc-operator
+      key: token
+  name: agent-pool-development
+  agentTokens:
+    - name: token-red
+```
 
-### Installation & Configuration
+The operator stores `token-red` agent token in a Kubernetes secret named TODO.
```

Comment:
@arybolovlev I'm having a hard time remembering what the secret will be named. Do you know what the secret name would be in this example? Thanks!

Reply:
In this example it will be `this-agent-pool`, where `this` is `metadata.name` value and `agent-pool` is the pre-defined value in our code.